server/src/test/resources/config/sauceConfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@mach6 thanks for pointing that out. it was checked in as part of new files. will remove it